buildspec.yml

```diff
@@ -47,7 +46,16 @@ phases:
           },{
             "source": "1.12.0-gpu",
             "dest": ["1.12.0-gpu", "1.12-gpu", "1.12.0-gpu-'${CODEBUILD_BUILD_ID#*:}'"]
-          }]
+          },{
+            "source": "1.13.0-cpu",
+            "dest": ["1.13.0-cpu", "1.13-cpu", "1.13.0-cpu-'${CODEBUILD_BUILD_ID#*:}'"]
+          },{
+            "source": "1.13.0-gpu",
+            "dest": ["1.13.0-gpu", "1.13-gpu", "1.13.0-gpu-'${CODEBUILD_BUILD_ID#*:}'"]
+          }],
+          "test": [
+            "tox -e py36 -- -n 8 test/integration/sagemaker/test_tfs.py --versions 1.13.0 --region {region} --registry 520713654638"
```

Comment:
we are updating all image versions, but only testing 1.13?

Reply:
i don't want this to timeout or run into the limit problem again. testing one version is already a big step forward since we didn't have any tests before. If this passes and the deployment didn't take too long. i will add the other tests. Does this sound ok to you?

Reply:
yes